js/components/Keyboard.tsx

```diff
@@ -209,13 +213,13 @@ export const Keyboard = ({ apple2, e }: KeyboardProps) => {
         />;
 
     const rows = keys.map((row, idx) =>
-        <div key={idx} className={`row row${idx}`}>
+        <div key={idx} className={cs(styles.row, styles[`row${idx}`])}>
             {row.map(bindKey)}
         </div>
     );
 
     return (
-        <div id="keyboard" style={{ marginLeft: e ? 0 : 15 }}>
+        <div className={styles.keyboard} style={{ marginLeft: e ? 0 : 15 }}>
```

Comment:
Couldn't the `marginLeft` now be moved into the module easily enough?

```jsx
className={cs([styles.keyboard, { styles.keyboard.e: e }])}
```

or similar?